FreeAPS/Sources/Modules/AddTempTarget/AddTempTargetStateModel.swift

```diff
@@ -189,5 +195,46 @@ extension AddTempTarget {
             }
             return Decimal(Double(target))
         }
+
+        func computePercentage(target: Decimal) -> Decimal {
+            let c = Decimal(hbt - 100)
```

Comment:
Consider using more descriptive variables names instead of `c`.

Reply:
For context, this is probably copied from oref0:

https://github.com/nightscout/trio-oref/blob/d1dfb7099d59e4f7779e5fc5c75f8ca299d91ef2/lib/determine-basal/determine-basal.js#L920-L926

Reply:
I used the “c” to be consistent with the other existing HBT computation. Is it ok to leave it like that for now (we will likely need to redo this any year in @mountrcg PR)

Reply:
Yes totally okay. I just wanted to note it 😊  The readability of the oref code is not really the best (as we know). No criticism on your work.